odl/tomo/backends/astra_setup.py

```diff
@@ -431,7 +430,7 @@ def astra_projection_geometry(geometry):
         raise ValueError('non-uniform detector sampling is not supported')
 
     if (isinstance(geometry, ParallelBeamGeometry) and
-            isinstance(geometry.detector, FlatDetector) and
+            isinstance(geometry.detector, (Flat1dDetector, Flat2dDetector)) and
```

Comment:
Well not really. It just says that the whole geometry is embedded in a 2D space, the detector can be anything. This was probably supposed to be the duck-typed version of `isinstance(geometry, Parallel2dGeometry)`.
But honestly if somebody decides to write a new geometry that doesn't subclass `Parallel2dGeometry` but still fulfills these conditions (whatever that may be), it would require a new conversion implementation anyway. So we might as well just do `isinstance(geometry, Parallel2dGeometry)`.